controllers/UsersController.js

```diff
@@ -0,0 +1,43 @@
+import status from '../helpers/StatusResponse';
```

Comment:
Import as Response which is being used in other parts of the application

Reply:
Thanks Joshua, I will refactor this immediately.